curator/defaults/client_defaults.py

```diff
@@ -14,7 +14,9 @@ def config_client():
         Optional('client_key', default=None): Any(None, str, unicode),
         Optional('aws_key', default=None): Any(None, str, unicode),
         Optional('aws_secret_key', default=None): Any(None, str, unicode),
-        Optional('aws_region', default=None): Any(None, str, unicode),
+        Optional('aws_token', default=None): Any(None, str, unicode),
+        Optional('aws_sign_request', default=False): Boolean(),
+        Optional('aws_region', default='us-east-1'): Any(None, str, unicode),
```

Comment:
Do you think it's wise to set the default to `us-east-1`, rather than just leave it kind of required to be set if you're using AWS credentials?

Reply:
Makes sense. I have removed the default.